archivist/archivist.py

```diff
@@ -550,7 +551,12 @@ def count(self, path: str, *, query: Optional[Dict] = None) -> int:
             headers=headers,
         )
 
-        return int(_headers_get(response.headers, HEADERS_TOTAL_COUNT))  # type: ignore
+        count = _headers_get(response.headers, HEADERS_TOTAL_COUNT)  # type: ignore
+
+        if count is None:
+            raise ArchivistHeaderError("Did not get a count in the header")
+
```

Comment:
update the doc string as well to document that this function can raise ArchivistHeaderError
(forgot to mention this yesterday)

see docstring for list() for example of format

Reply:
Added docstring

Reply:
excellent